GOVERNANCE.md

```diff
@@ -1,114 +1,295 @@
-
-## Governance
-
-### Operations
-The Operator Framework community is run and managed by the [community chairs][OWNERS_ALIASES].
-
-Note: The referenced names and chair positions will be edited in-place as chairs are added, removed, or replaced.
-
-A dedicated git repository will be the authoritative archive for membership list, code, documentation, and decisions made. The repository, along with this charter, will be hosted at github.com/operator-framework/community.
-
-The mailing list at groups.google.com/forum/#!forum/operator-framework will be used as a place to call for and publish group decisions, and to hold discussions in general.
-
-### Operator Framework Community Membership
-All active members of the Operator Framework community are listed in the [`OWNERS`][OWNERS_ALIASES] or in the OWNERS files in Operator Framework subprojects.
-
-New members can apply for membership by creating an Issue or Pull Request on the repository on GitHub indicating their desire to join.
-
-Membership can be surrendered by creating an Issue stating this desire, or by creating a Pull Request to remove the own name from the members list.
-
-### Decision Process
-This group will seek consensus decisions. After public discussion and consideration of different opinions, the Chair and/or Co-Chair will record a decision and summarize any objections.
-
-All community members who have joined the GitHub group at least 21 days prior to the vote are eligible to vote. This is to prevent people from rallying outside supporters for their desired outcome.
+* Community over Product or Company: Sustaining and growing our community takes
+    priority over shipping code or sponsors' organizational goals. Each
+    contributor participates in the project as an individual.
 
-When the group comes to a decision in a meeting, the decision is tentative. Any group participant may object to a decision reached at a meeting within 7 days of publication of the decision on the GitHub Issue and/or mailing list. That decision must then be confirmed on the GitHub Issue via a Call for Agreement.
+* Inclusivity: We innovate through different perspectives and skill sets, which
+    can only be accomplished in a welcoming and respectful environment.
 
-The Call for Agreement, when a decision is required, will be posted as a GitHub Issue or Pull Request and must be announced on the mailing list. It is an instrument to reach a time-bounded lazy consensus approval and requires a voting period of no less than 7 days to be defined (including a specific date and time in UTC).
+* Participation: Responsibilities within the project are earned through
+    participation, and there is a clear path up the contributor ladder into leadership
+    positions.
```

Comment:
+consensus seeking

Reply:
when discussed in the office hour, it was determined that consensus seeking is part of being inclusive